### PR TITLE
Revert Halloween theme

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Reverting of https://github.com/cowprotocol/cowswap/commit/e9a27ac172054510a58f60f5527ec840e8c5b552

  # To Test

With the dark mode enabled:
1. The logo should be default (no spider rag)
2. Pending, submitted Moo sounds should be default

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]

**Review by anxolin — COMMENTED** [review]
All good, only some minor comment

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Set up node
         uses: actions/setup-node@v2
         with:
-          node-version: lts/*
+          node-version: 16.17.1
```
[the remark this answers is not in the document]
Comment (alfetopito) [reply .github/workflows/ci.yml:35]: I am currently dealing with GH action and will take care of that

Unless you want something now then feel free to go ahead and update it

[the remark this answers is not in the document]
Comment (anxolin) [reply .github/workflows/ci.yml:35]: no, if you address it in another branch is fine with me

[the remark this answers is not in the document]
Comment (alfetopito) [reply .github/workflows/ci.yml:35]: Still working on it, but here's the draft PR https://github.com/cowprotocol/cowswap/pull/1268/files

**Pull request merged**